addon-test-support/properties/click-on-text.js

```diff
@@ -97,9 +97,9 @@ export function clickOnText(selector, userOptions = {}) {
           let fullSelector = buildSelector(this, context, selector, options);
           let container = options.testContainer || findClosestValue(this, 'testContainer');
 
-          context.assertElementExists(fullSelector, options);
+          const $ele = context.assertElementExists(fullSelector, options);
 
-          return context.click(fullSelector, container, options);
+          return context.click($ele[0], container, options);
```

Comment:
This is technically a breaking change, unfortunatelly. At the moment execution contexts are a bit inconsistent. Some of them(RFC268, NativeEvents), do make sure they do only operate with a single element,
while the rest(Acceptance, Integration) just accept a selector and then pass it to a certain action helper.
you can compare it by looking at
https://github.com/san650/ember-cli-page-object/blob/5814ba4a82b672dd899b0c4dac25355be01b16a6/addon-test-support/-private/execution_context/native-events-context.js#L39
and 
https://github.com/san650/ember-cli-page-object/blob/5814ba4a82b672dd899b0c4dac25355be01b16a6/addon-test-support/-private/execution_context/acceptance.js#L38

While I believe actions against multiple elements is bad, I think we can't just break this behavior w/o major version bump. And that's one of the primary point for v2(#479).

At the moment, the best way I can think of to achieve that behavior in v1 may be to introduce a new click(`clickElement`?) method for each execution context, which would accept just an element. It would also require a special code path in the click-on-text itself for `matchExact` only. And that all would be removed in favour of a more concise handling of `matchExact` in v2.

thoughts?